crates/nu-cli/src/repl.rs

```diff
@@ -131,16 +131,13 @@ pub fn evaluate_repl(
         // escape a few things because this says so
         // https://code.visualstudio.com/docs/terminal/shell-integration#_vs-code-custom-sequences-osc-633-st
         let cmd_text = line_editor.current_buffer_contents().to_string();
+        let len = cmd_text.len();
+        let mut cmd_text_chars = cmd_text[0..len].chars();
```

Comment:
This slicing seems unnecessary, should be the same as the `cmd_text` as `&str`

Reply:
agreed.